packages/devtools/lib/src/main.dart

```diff
@@ -30,21 +31,14 @@ import 'utils.dart';
 const bool showMemoryPage = false;
 const bool showPerformancePage = false;
 
+const flutterLibraryUriCandidates = [
+  'package:flutter/src/widgets/binding.dart',
```

Comment:
nit: can you use a public package: url for this?
For example:
package:flutter/widgets.dart'